src/pages/Home.tsx

```diff
@@ -1,29 +1,99 @@
-import React, {useState} from 'react';
+import React, {useState, useEffect} from 'react';
 import Map from '../components/Map/Map';
-import {Text, Heading, Box, Flex, Spacer} from '@chakra-ui/react';
+import {Text, Heading, Box, Flex, Spacer, IconButton} from '@chakra-ui/react';
 import AqiDial from '../components/AqiGauge/AqiDial';
 import {useTranslation} from 'react-i18next';
+import {ChevronDownIcon, ChevronUpIcon} from '@chakra-ui/icons';
 
 /**
  * Home screen component
  */
 const Home: () => JSX.Element = () => {
   // State for which sensor to display in the current sensor box
   const [currentSensor, setCurrentSensor] = useState('');
+  const [isMobile, setIsMobile] = useState(
+    window.matchMedia('(max-width: 47.9em)')?.matches ?? false
+  );
+  const [showGraphUI, setShowGraphUI] = useState(false);
+  const [showGaugeUI, setShowGaugeUI] = useState(false);
+  const [showMapUI, setShowMapUI] = useState(true);
 
   const {t} = useTranslation('home');
 
+  // -------- Detect screen size for conditional formatting --------- //
+  /** Adjust UI for switching between mobile and desktop modes */
+  function handleScreenChange(this: MediaQueryList): void {
+    // Is the screen size mobile size
+    if (this.matches) {
+      setIsMobile(true);
+    } else {
+      setIsMobile(false);
+    }
+  }
+
+  // Updates the state and the dom when the window size is changed
+  useEffect(() => {
+    const screenSize = window.matchMedia('(max-width: 47.9em)');
+    if (screenSize) {
+      screenSize.addEventListener('change', handleScreenChange);
+    }
+
+    return function (): void {
+      if (screenSize) {
+        screenSize.removeEventListener('change', handleScreenChange);
+      }
+    };
+  }, []);
+  // -----------------  End detect screen size ----------------- //
+
   return (
     <Box>
       <Text>{t('constructionNotice')}</Text>
-      <Flex direction={['column', 'column', 'row', 'row']} marginTop={4}>
-        <Box flex="2" marginX={4} height={['100%']}>
-          <Map updateCurrentSensor={setCurrentSensor} />
-        </Box>
+      <Flex direction={['column', 'column', 'row', 'row']} textAlign="center">
+        {isMobile ? (
```

Comment:
I think it might be easier to read too if there were one `isMobile` ternary operator that had all of the mobile versions of each component rather than 3 `isMobile` checks for the 3 different subcomponents of the home page

Reply:
Opened issue for this